frontend/src/components/home-page/question-types/multiple-choice.tsx

```diff
@@ -29,27 +29,33 @@ const AnswerChoice = ({
   return (
     <div
       className={`answer-choice ${
-        showCorrectPreviewResponse ? (correct ? "correct" : "incorrect") : ""
```

Comment:
Can you split this file into two separate files? One for the `MultipleChoice` component and one for the `AnswerChoice` component? We shouldn't combine components in the same file, even if they are used once.

Reply:
i wanna combine them into one component tbh :cake:

Reply:
That works too